docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,76 @@
+# Deploy SD-Core with CUPS
+
+This guide covers how to install a multi-node SD-Core with Control Plane and User Plane Separation (CUPS).
+
+## Requirements
+
+- Juju >= 3.1
+- A Juju controller has been bootstrapped, and is externally reachable
+- A Control Plane Kubernetes cluster configured with
+  - 1 available IP address for the Access and Mobility Management Function (AMF)
+- A User Plane Kubernetes cluster configured with
+  - 1 available IP address for the User Plane Function (UPF)
```

Comment:
For sdcore user plane preparing Macvlan bridges and network interfaces are missing.

Reply:
Added to L14